deploy/dev/values.yaml

```diff
@@ -80,6 +80,11 @@ indexers:
     parameters:
       block_height_start: 1275799
       rpc_thread_blocks: 1
+  - network: ethereum
+    worker: lido
+    endpoint: <path:kv/data/network/rss3-node#ENDPOINT_ETHEREUM>
+    parameters:
+      block_number_start: 17571053
```

Comment:
Why is block number `17571053`?

Reply:
fixed, using 11473216 for now ( `AddressStakedETH (0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84)` 's deploy block height)